### ISSUE TITLE
About page tabs cannot be switched with keyboard

### ISSUE THREAD

**Issue by caugner** [mdn/fred#745]
STR:

1. Go to https://developer.mozilla.org/en-US/about
2. Press <kbd>Tab</kbd> until you focus the first tab
3. Try to change the tab

Expected: to be able to change the tab

Actual: the tab cannot be changed

Note: Observatory also has tabs ([example](https://developer.mozilla.org/en-US/observatory/analyze?host=mdn.dev#scoring)), but keyboard navigation works there.
